layouts/partials/head.html

```diff
@@ -16,8 +16,8 @@
 <meta name="robots" content="noindex, nofollow" />
 {{ end -}} {{ partialCached "favicons.html" . }}
 <title>
-    {{- if .IsHome -}} {{ .Site.Title -}} {{ else -}} {{ with .Title }}{{ . }} |
-    {{ end -}} {{ .Site.Title -}} {{ end -}}
+    {{- if .IsHome -}} {{ .Site.Title -}} {{ else -}} {{ with .Title }}{{ . }}
+    {{ end -}} {{ end -}}
```

Comment:
Unsure: without `{{ .Site.Title -}}` do we still need both instances of `{{ end -}}`?

Reply:
Yep - they're for the if and the with clause as far as I understand